unity-client/Assets/Scripts/MainScripts/DCL/Environment/MessagingContext/MessagingContext.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace DCL
+{
+    public class MessagingContext : System.IDisposable
+    {
+        public readonly MessagingControllersManager messagingControllersManager;
```

Comment:
When we split `SceneController` in different classes (`MessageBridge`, `MessageQueuer`, `MessageProcessor`, etc). This context will grow,